sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FlatMapGroupsWithStateExec.scala

```diff
@@ -111,28 +111,41 @@ case class FlatMapGroupsWithStateExec(
       indexOrdinal = None,
       sqlContext.sessionState,
       Some(sqlContext.streams.stateStoreCoordinator)) { case (store, iter) =>
+        val numOutputRows = longMetric("numOutputRows")
```

Comment:
Just wondering what's the difference between this and the metric with the same name in `InputProcessor`? Don't they collide somehow?

Reply:
ping @HeartSaVioR

Reply:
Sorry missed this. I accidentally marked this as resolved at mobile so rolled back. Will get back when I got time.

Reply:
You're right I missed the metrics in InputProcessor. We can remove `numOutputRows` here. Thanks for finding missing spot!

Reply:
Addressed.